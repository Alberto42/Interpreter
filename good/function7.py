def f(x) {
    if (x > 0) {
        return x + f(x-1)
    }
    return 0
}

x = f(5)