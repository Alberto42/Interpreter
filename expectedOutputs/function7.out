Right (OK,
Ident "x" = VInt 15
)
